hudi-common/src/test/java/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderBase.java

```diff
@@ -155,13 +155,14 @@ public void testReadLogFilesOnlyInMergeOnReadTable(RecordMergeMode recordMergeMo
     // Use InMemoryIndex to generate log only mor table
     writeConfigs.put("hoodie.index.type", "INMEMORY");
 
+
     try (HoodieTestDataGenerator dataGen = new HoodieTestDataGenerator(0xDEEF)) {
-      // One commit; reading one file group containing a base file only
-      commitToTable(dataGen.generateInserts("001", 100), INSERT.value(), writeConfigs);
+      // One commit: reading one file group containing 1 log file only.
```

Comment:
I did not review entire test class. but can you confirm we are not giving up on straight forward test case, where 1st commit added a base file right?